packages/online-shell/webpack.config.dev.js

```diff
@@ -227,7 +214,12 @@ module.exports = () => {
         // for accessing /login from the app. Having it here allows
         // the paths above to remain external to the app
         //
-        const history = historyApiFallback()
+        const history = historyApiFallback({
+          verbose: true,
+          index: '/',
+        })
+
+        console.log(history)
```

Comment:
do we want to have console.log ? or is it just a leftover

Reply:
leftover. Will remove.